### ISSUE TITLE
 Incompatibility with Strapi version 4.15.2, 4.15.4

### ISSUE THREAD

**Issue by AmmonLin** [plantagoIT/strapi-drag-drop-content-type-plugin#58]
# Environment

-    Package Version: @retikolo/drag-drop-content-types@1.3.10
-    Node.js Version: 20.9.0
-    Strapi Version: 4.15.2, 4.15.4 

# Description

I am encountering an issue after trying to install @retikolo/drag-drop-content-types@1.3.10 using Strapi version 4.15.4. 

# Steps to Reproduce

1. using `npx create-strapi-app@4.15.4 my-project` to create a project and select `Quickstart`.
2. cd to `my-project` and then install plugin `npm install @retikolo/drag-drop-content-types`
3. build the project `npm run build`

# Expected Behavior

Build success without error.

# Actual Behavior

The build process fail with following error message:

>  ⠋ Building admin panel[ERROR] Module not found: Error: Package path ./admin/src/content-manager/pages/ListView/actions is not exported from package ...

**Comment by derschiw** [comment]
Yes I notice the same problem. Unfortunately I don't now how to fix it. It seems like this line is not working anymore. Anyone an idea?

```
import { getData, getDataSucceeded } from "@strapi/admin/admin/src/content-manager/pages/ListView/actions";
```

**Comment by derschiw** [comment]
I asked in the strapi forum for some help 
https://forum.strapi.io/t/cannot-use-listview-actions-since-4-15-in-plugin/33652

**Comment by derschiw** [comment]
@jpizzle34 since you implemented this a while ago, do you have any idea how to fix this?

**Comment by derschiw** [comment]
Its now WIP in https://github.com/plantagoIT/strapi-drag-drop-content-type-plugin/pull/59

**Comment by derschiw** [comment]
There is a hotfix now, which allows the plugin to run. However it's not very nice, since it causes a page-reload after every sorting action. 

I am currently working on fixing this issue properly, but since Strapi removed direct imports, I'm not sure if I can find a work-around. I'll update this, until then the discussion is in #59

**Comment by Boegie19** [comment]
@derschiw I asume the reason you where using import { getData, getDataSucceeded } from "@strapi/admin/admin/src/content-manager/pages/ListView/actions" is to reload the content witout reloading the full page?

**Comment by derschiw** [comment]
@Boegie19 yes, that is to only reload only the list (without loading the full page)

What would be the work-around? I'd very much appreciate it 😊

**Comment by Boegie19** [comment]
in theory strapi's you have access to all strapi providers declarer in https://github.com/strapi/strapi/blob/a549feb253588d8b74a2de6d9c9eb0d5ea46702c/packages/core/admin/admin/src/components/Providers.tsx

aka you have access to the redux store so in most ceases You could make the change manually second option would maby be to add a filter somehow since that would also trigger a partial refresh.

**Comment by jpizzle34** [comment]
> @jpizzle34 since you implemented this a while ago, do you have any idea how to fix this? Your help would be very nice! :)

Hi @derschiw I'll have a look. It's been a while so I'll catch up with the project first :)

**Comment by derschiw** [comment]
Fixed with #64
